content/hi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,215 @@
+---
+title: क्लाउड नियंत्रक प्रबंधक
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state state="beta" for_k8s_version="v1.11" >}}
+
+क्लाउड इंफ्रास्ट्रक्चर टेक्नोलॉजी आपको सार्वजनिक, निजी और हाइब्रिड क्लाउड पर कुबेरनेट चलाने देती है।
+कुबेरनेट्स स्वचालित, एपीआई-संचालित बुनियादी ढांचे के बीच तंग युग्मन के बिना विश्वास करता है
+अवयव।
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
```

Comment:
For `glossary_definition` to work as expected we should localize the glossary entry for "cloud-controller-manager" (i.e. [en/docs/reference/glossary/cloud-controller-manager.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/cloud-controller-manager.md)) and incorporate it into this pull request by adding to list of Hindi glossary items inside [/hi/docs/reference/glossary/cloud-controller-manager.md](https://github.com/kubernetes/website/tree/main/content/hi/docs/reference/glossary/).

Reply:
@akash-kumar-saw As part of this pull request, you would also need to add the Hindi translation for the glossary entry `cloud-controller-manager` to complete the localization of this 'Cloud Controller Manager' page.